sql/triggers.sql

```diff
@@ -64,10 +70,11 @@ BEGIN
 
   FOR rec IN EXECUTE query LOOP
     changes := row_to_json(rec);
+    previous := '{}'::jsonb;
```

Comment:
I'm not sure if it makes sense to use an empty json object as the previous value here, maybe it should be NULL instead?